management/server/peer.go

```diff
@@ -971,3 +973,28 @@ func (am *DefaultAccountManager) updateAccountPeers(ctx context.Context, account
 
 	wg.Wait()
 }
+
+func (am *DefaultAccountManager) processGetAccountRequests(ctx context.Context) {
+	for {
+		select {
+		case req := <-am.getAccountRequestCh:
+			am.mu.Lock()
+			am.getAccountRequests[req.AccountID] = append(am.getAccountRequests[req.AccountID], req)
+			if len(am.getAccountRequests[req.AccountID]) == 1 {
+				go func(ctx context.Context, accountID string) {
+					bufferIntervalStr := os.Getenv("NB_GET_ACCOUNT_BUFFER_INTERVAL")
```

Comment:
The env vars will not change. Move the bufferInterval initializastion to an init() function or into a constructor.